src/pivot-table/hooks/use-selections-model.ts

```diff
@@ -66,7 +70,11 @@ export default function useSelectionsModel(
   );
 
   const select = useCallback(
-    (qType: EngineAPI.NxSelectionCellType, qRow: number, qCol: number) => async () => {
+    (qType: EngineAPI.NxSelectionCellType, qRow: number, qCol: number) => async (evt: React.MouseEvent) => {
+      if ((evt.target as HTMLElement)?.className.includes("sn-pivot-table-column-adjuster")) {
```

Comment:
Not sure if this is the place to check for this, but need to cancel selection if the event comes from the adjuster